static/templates/draft.handlebars

```diff
@@ -9,9 +9,9 @@
                 </div>
 
                 <span class="stream_topic">
-                    <div class="message_label_clickable narrows_by_subject">
-                        {{topic}}
-                    </div>
+                    <span class="message_label_clickable narrows_by_subject">
+                        {{~ topic ~}}
```

Comment:
Just out of curiosity, how does the `~` work in this case?

Reply:
You can read the conversation I had with @andersk for context :) https://chat.zulip.org/#narrow/stream/9-issues/subject/double.20topic.3F/near/621828

Reply:
Cool. Thanks! Didn't know about this before.